Source/WebCore/platform/graphics/GraphicsLayer.cpp

```diff
@@ -197,6 +197,9 @@ GraphicsLayer::GraphicsLayer(Type type, GraphicsLayerClient& layerClient)
     , m_backfaceVisibility(true)
     , m_masksToBounds(false)
     , m_drawsContent(false)
+#if HAVE(HDR_SUPPORT)
+    , m_drawsHDRContent(false)
```

Comment:
nit: would be nice to inline in the header. As far as I know this works for bitfields with modern C++.

Reply:
Fixed.